modular_skyrat/modules/modular_implants/code/nifsofts/huds.dm

```diff
@@ -0,0 +1,141 @@
+/datum/nifsoft/hud
+	name = "blank HUD"
+	program_desc = "Provides the overlay of a hud on compatible eyewear."
```

Comment:
NIF Scrying Lens

An umbrella term for all sorts of NIFsofts dealing with heads-up displays, this sort of technology dates back almost to the beginning of NIFsoft development. These 'softs are commonly used in the civilian field for integration with all sorts of activities; piloting, swordplay, scientific research, or even AI copiloting for important social interactions. While normally the nanomachines involved in the program's operation are used as a sort of artificial contact lens over the user's visual organs, NanoTrasen regulations have bid these particular forks to instead integrate with glasses the user's already wearing.